services-js/commissions-app/src/client/common/Checkbox.stories.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Checkbox from './Checkbox';
+
+storiesOf('Checkbox', module).add('default', () => (
+  <Checkbox name="checkbox" title="" value="checkbox" onChange="" onBlur="" />
```

Comment:
Should add a title here so that you get a better idea of what the component looks like in Storybook.

Remember that you’ll need to re-run the tests with `-u` to pick up snapshot changes.